independent-projects/tools/pom.xml

```diff
@@ -50,7 +50,7 @@
         <version.surefire.plugin>3.0.0-M5</version.surefire.plugin>
         <nexus-staging-maven-plugin.version>1.6.8</nexus-staging-maven-plugin.version>
         <quarkus.version>999-SNAPSHOT</quarkus.version>
-        <maven-model-helper.version>16</maven-model-helper.version>
+        <maven-model-helper.version>17</maven-model-helper.version>
```

Comment:
18 is coming up and will be updated in https://github.com/quarkusio/quarkus/pull/14498

Reply:
@gastaldi cool :) anyway this slipped in that PR (and shouldn't be here) 😂